vcluster/reference/migrations/loft-cli-vcluster-cli-migration.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Loft CLI to vCluster CLI Migration
+sidebar_label: Loft CLI Migration
+sidebar_position: 2
+description: Overview of loft CLI commands that have been moved to vcluster CLI
+---
+
+This section will give an overview over the loft CLI commands that have moved to vcluster CLI.
+Most commands should work exactly as before. Any commands that have been changed are documented in the "Comments" column.
+
+## Global --config flag
+
+Previously, **loft** CLI read its config file from `~/.loft/config.json` if nothing was specified under `--config`. **vcluster** CLI now assumes the config to be in `~/.vcluster/config.json` if nothing is specified under `--config`.
+Futhermore, the config format in **vcluster** CLI changed in a way that the loft platform (now vCluster platform) related configuration goes into a special section under the key `platform`.
+If you want you can convert your already existing loft configuration via the following (prerequisite beeing jq >= 1.7 installed):
```

Comment:
Is this supposed to have `beeing` or a different word?

Reply:
Dropped it.